packages/transaction-controller/src/TransactionController.ts

```diff
@@ -183,6 +188,32 @@ export const CANCEL_RATE = 1.5;
  */
 export const SPEED_UP_RATE = 1.1;
 
+/**
+ * @type IncomingTransactionOptions
+ *
+ * Configuration options for the IncomingTransactionHelper
+ * @property includeTokenTransfers - Whether or not to include ERC20 token transfers.
+ * @property isEnabled - Whether or not incoming transaction retrieval is enabled.
+ * @property queryEntireHistory - Whether to initially query the entire transaction history or only recent blocks.
+ * @property updateTransactions - Whether to update local transactions using remote transaction data.
+ */
+type IncomingTransactionOptions = {
```

Comment:
If these are part of the constructor, then by extension should we `export` these so they can be referenced externally?

Reply:
works for me

Reply:
https://github.com/MetaMask/core/pull/3876